rogue/src/main.rs

```diff
@@ -536,13 +540,21 @@ impl GameState for State {
             }
             RunState::HazardTurn => {
                 self.run_hazard_turn_systems();
+                let get_blessing = is_player_encroaching_blessing_tile(&self.ecs)
```

Comment:
How dare you call these blessings a hazard.

I assume `RunState::HazardTurn` is being used as more of a catch-all for hazards+boons here? Is the idea that you don't want e.g. monsters to get a turn while interacting with a blessing statue?

Reply:
If I recall: each turn is split up into three "subturns": the player acts, monsters act, then "hazards" act. My initial hazards were like, fire which spreads and evaporates, hence the name "hazard".

I'm gonna guess I got a bit lazy here and reused the hazard subturn to run the blessing systems.